spec/index.bs

```diff
@@ -11516,6 +11524,15 @@ interface mixin GPURenderCommandsMixin {
 
     undefined drawIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
     undefined drawIndexedIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
+
+    undefined multiDrawIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset,
+        GPUSize32 maxDrawCount,
+        optional GPUBuffer drawCountBuffer = null,
```

Comment:
from @kainino0x: nit for editors: need to check if these GPUBuffers need to be nullable

Reply:
To match feature parity with D3D12 and Vulkan the `drawCountBuffer` should be nullable, in which case `maxDrawCount` becomes the draw count.

Reply:
I think the question was about optional vs nullable